test/hash_table_tests/Makefile

```diff
@@ -0,0 +1,38 @@
+SHELL           = /bin/sh
```

Comment:
Are we going back to make? I thought that everything should be `cmake` now, and all build scripts are `cmake` in both ccpp-physics and ccpp-framework at the moment.

Reply:
That sounds like a good idea. It is so simple, I never considered doing the extra work to create a CMakeLists.txt file.